sync_diff_inspector/dm_test.go

```diff
@@ -0,0 +1,165 @@
s]]\\n  schema-pattern = \\\"sharding*\\\"\\n  table-pattern = \\\"t*\\\"\\n  target-schema = \\\"db_target\\\"\\n  target-table = \\\"t_target\\\"\\n\\n[[route-rules]]\\n  schema-pattern = \\\"sharding*\\\"\\n  table-pattern = \\\"\\\"\\n  target-schema = \\\"db_target\\\"\\n  target-table = \\\"\\\"\\n\\n[[mapping-rule]]\\n  schema-pattern = \\\"sharding*\\\"\\n  table-pattern = \\\"t*\\\"\\n  source-column = \\\"id\\\"\\n  target-column = \\\"id\\\"\\n  expression = \\\"partition id\\\"\\n  arguments = [\\\"1\\\", \\\"sharding\\\", \\\"t\\\"]\\n  create-table-query = \\\"\\\"\\n\\n[block-allow-list]\\n  do-dbs = [\\\"~^sharding[\\\\\\\\d]+\\\"]\\n\\n  [[block-allow-list.do-tables]]\\n    db-name = \\\"~^sharding[\\\\\\\\d]+\\\"\\n    tbl-name = \\\"~^t[\\\\\\\\d]+\\\"\\n\",\"is-sharding = true\\nshard-mode = \\\"pessimistic\\\"\\nonline-ddl-scheme = \\\"\\\"\\ncase-sensitive = false\\nname = \\\"test\\\"\\nmode = \\\"all\\\"\\nsource-id = \\\"mysql-replica-02\\\"\\nserver-id = 0\\nflavor = \\\"\\\"\\nmeta-schema = \\\"dm_meta\\\"\\nheartbeat-update-interval = 1\\nheartbeat-report-interval = 10\\nenable-heartbeat = false\\ntimezone = \\\"Asia/Shanghai\\\"\\nrelay-dir = \\\"\\\"\\nuse-relay = false\\nfilter-rules = []\\nmydumper-path = \\\"./bin/mydumper\\\"\\nthreads = 4\\nchunk-filesize = \\\"64\\\"\\nstatement-size = 0\\nrows = 0\\nwhere = \\\"\\\"\\nskip-tz-utc = true\\nextra-args = \\\"\\\"\\npool-size = 16\\ndir = \\\"./dumped_data.test\\\"\\nmeta-file = \\\"\\\"\\nworker-count = 16\\nbatch = 100\\nqueue-size = 1024\\ncheckpoint-flush-interval = 30\\nmax-retry = 0\\nauto-fix-gtid = false\\nenable-gtid = false\\ndisable-detect = false\\nsafe-mode = false\\nenable-ansi-quotes = false\\nlog-level = \\\"\\\"\\nlog-file = \\\"\\\"\\nlog-format = \\\"\\\"\\nlog-rotate = \\\"\\\"\\npprof-addr = \\\"\\\"\\nstatus-addr = \\\"\\\"\\nclean-dump-file = true\\n\\n[from]\\n  host = \\\"127.0.0.1\\\"\\n  port = 3307\\n  user = \\\"root\\\"\\n  password = \\\"/Q7B9DizNLLTTfiZHv9WoEAKamfpIUs=\\\"\\n  max-allowed-packet = 67108864\\n\\n[to]\\n  host = \\\"127.0.0.1\\\"\\n  port = 4000\\n  user = \\\"root\\\"\\n  password = \\\"\\\"\\n  max-allowed-packet = 67108864\\n\\n[[route-rules]]\\n  schema-pattern = \\\"sharding*\\\"\\n  table-pattern = \\\"t*\\\"\\n  target-schema = \\\"db_target\\\"\\n  target-table = \\\"t_target\\\"\\n\\n[[route-rules]]\\n  schema-pattern = \\\"sharding*\\\"\\n  table-pattern = \\\"\\\"\\n  target-schema = \\\"db_target\\\"\\n  target-table = \\\"\\\"\\n\\n[[mapping-rule]]\\n  schema-pattern = \\\"sharding*\\\"\\n  table-pattern = \\\"t*\\\"\\n  source-column = \\\"id\\\"\\n  target-column = \\\"id\\\"\\n  expression = \\\"partition id\\\"\\n  arguments = [\\\"2\\\", \\\"sharding\\\", \\\"t\\\"]\\n  create-table-query = \\\"\\\"\\n\\n[block-allow-list]\\n  do-dbs = [\\\"~^sharding[\\\\\\\\d]+\\\"]\\n\\n  [[block-allow-list.do-tables]]\\n    db-name = \\\"~^sharding[\\\\\\\\d]+\\\"\\n    tbl-name = \\\"~^t[\\\\\\\\d]+\\\"\\n\"]}")
```

Comment:
😵how about using backquote string

Reply:
just copy from dm's log, ignore it :cold_sweat:

Reply:
seems I don't have permission to resolve comments in this repo 🤔

Reply:
I remove this conversation